bluepyemodel/evaluation/fitness_calculator_configuration.py

```diff
@@ -117,6 +120,8 @@ def __init__(
                         {"name": str, "stimuli": list of dict, "recordings": list of dict,
                         "validation": bool}
                     ]
+
+                here, recording_name should have the format ``f"{location}.v"``, e.g. ``"soma.v"``
```

Comment:
Do you mean the recording_name in one of the dictionaries of recordings?

Reply:
Yes, indeed it is not clear. Let me change the phrasing